bedrock/firefox/templates/firefox/channel/ios.html

```diff
@@ -6,7 +6,21 @@
 
 {% add_lang_files "firefox/channel/index" %}
 
-{% block page_desc %}{{_('Test beta versions of Firefox for iOS via Apple’s TestFlight program and help make our mobile browser for iPhone, iPad and iPod touch even better.')}}{% endblock %}
+{% block page_title %}
+{% if l10n_has_tag('firefox_title_meta_bug1345338') %}
+  {{ _('Try New Features in a Pre-Release iOS Browser | Firefox') }}
+{% else %}
+  {{_('Test beta versions of Firefox for iOS via Apple’s TestFlight program and help make our mobile browser for iPhone, iPad and iPod touch even better.')}}
```

Comment:
The title and the description contain the same text: Test beta versions of Firefox for iOS ...